packages/cpp/src/node/cpp-contribution.ts

```diff
@@ -49,4 +49,11 @@ export class CppContribution extends BaseLanguageServerContribution {
         this.forward(clientConnection, serverConnection);
     }
 
+    protected onDidFailSpawnProcess(error: Error): void {
+        super.onDidFailSpawnProcess(error);
+        console.error("Error starting C/C++ language server.");
```

Comment:
I think that using the `Logger` defined [here](https://github.com/theia-ide/theia/blob/master/packages/core/src/common/logger.ts) is now the correct way to log stuff in Theia. There are a lot of examples like [this one](https://github.com/theia-ide/theia/blob/master/packages/terminal/src/node/shell-terminal-server.ts#L21) to get you started.

Although it might be confusing as the class `CppContribution` extends (`BaseLanguageServerContribution`) seems to be using the `console` object. @hexa00 maybe we should try to remove all the references to console and replace them with logger as to avoid confusion (in another pr of course)

Reply:
The `BaseLanguageServerContribution` should use the logger

Reply:
I changed this patch to use the logger.